### DISCUSSION TITLE
제가 처음 만든 프로젝트 파일을 배포합니다 :D [치자피즈]

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#365] [4. General]
월드에 도전하시는 크리에이터 분들을 응원합니다!

처음 시작하시는 분께는 소소한 도움이 되지는 않을까 프로젝트 파일을 배포하려고 해요.

이곳에서 받아주세요 :D
https://cheesafizz.notion.site/14cfd361391d4325b2ded863957f94be

**Comment by 17beans** [comment]
안녕하세요 윈도우 PC 사용자입니다.

압축을 해제하실 때 관리자 권한으로 압축을 해제하셔야 프로젝트 폴더를 정상적으로 이용 가능하니 윈도우 PC 사용자분들께서는 참고해주시기 바랍니다.🙂

**Comment by rjc90** [comment]
감사합니다!!!

**Reply by srlee010** [reply]
감사합니다 톡드렸어요!
